### PR TITLE
SG-22646 Replaced references to Shotgun with ShotGrid

### PR DESCRIPTION
**Pull request by eshokrgozar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@
 [![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
 
 ## Documentation
-This repository is a part of the Shotgun Pipeline Toolkit.
+This repository is a part of the ShotGrid Pipeline Toolkit.
 
 - For more information about this app and for release notes, *see the wiki section*.
 - For general information and documentation, click here: https://support.shotgunsoftware.com/entries/95441257
-- For information about Shotgun in general, click here: http://www.shotgunsoftware.com/toolkit
+- For information about ShotGrid in general, click here: https://help.autodesk.com/view/SGSUB/ENU/
```
Comment (jfboismenu) [inline README.md:11]: Could you update the rebrand.py script with the required regex and send me a copy? Everyone can rerun the script on their branches.

**Review by jfboismenu — COMMENTED** [review]
👍

**Comment by jfboismenu** [comment]
@mathurf , your turn!

**Review by mathurf — CHANGES_REQUESTED** [review]
tk-aftereffects/hooks/tk-multi-publish2/basic/publish_document.py should be updated

**Review by mathurf — APPROVED** [review]
Everything looks good now

**Pull request merged**